### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by ccoVeille**
- **trim all lines and fix double spaces**
- **fix typos**
- **fix trademark and acronym**

### CODE REVIEW

**Comment by ccoVeille** [comment]
Please accept my submission, I noticed a lot of typos at first, then I fixed brands and acronyms

File: data/apps.csv [hunk 1]
```diff
@@ -429,7 +429,7 @@ webdev,lychee,,https://github.com/lycheeverse/lychee,"Fast, async, resource-frie
 networking,Rustcat,,https://github.com/robiot/rustcat,Netcat Alternative in Rust.
 funny,ponysay,,https://github.com/erkin/ponysay,Pony rewrite of cowsay.
 funny,yosay,,https://github.com/yeoman/yosay,"Like cowsay, but for yeoman."
-graphics,cli-mandelbrot,,https://github.com/danyshaanan/cli-mandelbrot,A cli for traversing the Mandelbrot fractal.
+graphics,cli-mandelbrot,,https://github.com/danyshaanan/cli-mandelbrot,A CLI for traversing the Mandelbrot fractal.
```
Comment (vitorhcl) [inline data/apps.csv:432]:
```suggestion
graphics,cli-mandelbrot,,https://github.com/danyshaanan/cli-mandelbrot,Traverse the Mandelbrot fractal.
```

File: data/apps.csv [hunk 2]
```diff
@@ -1100,7 +1100,7 @@ online,Fjira,,https://github.com/mk-5/fjira,The fuzziest Jira command line tool
 networking,neoss,,https://github.com/PabloLec/neoss,User-friendly and detailed socket statistics with a Terminal UI.
 conversion,unoserver,,https://github.com/unoconv/unoserver,"Using LibreOffice as a server for converting documents, it allows to convert multiple documents without loading libreoffice into memory every time."
 security,gocryptfs,https://nuetzlich.net/gocryptfs,https://github.com/rfjakob/gocryptfs,An encrypted overlay filesystem written in Go.
-conversion,jsonify-resume,,https://github.com/ashishbinu/jsonify-resume,A cli that converts resumes into JSON Resume schema.
+conversion,jsonify-resume,,https://github.com/ashishbinu/jsonify-resume,A CLI that converts resumes into JSON Resume schema.
```
Comment (vitorhcl) [inline data/apps.csv:1103]:
```suggestion
conversion,jsonify-resume,,https://github.com/ashishbinu/jsonify-resume,Convert resumes into JSON Resume schema.
```

File: data/apps.csv [hunk 3]
```diff
@@ -1125,7 +1125,7 @@ data-management-json,jp,,https://github.com/jmespath/jp,"A command line interfac
 data-management-json,JSON Command,,https://github.com/zpoley/json-command,JSON command line processing toolkit: no more writing code to inspect or transform JSON objects.
 data-management-json,jsonv.sh,,https://github.com/archan937/jsonv.sh,A Bash command line tool for converting JSON to CSV.
 data-management-json,JSON-Grep,,https://github.com/ploubser/JSON-Grep,JGrep is a command line tool and API for parsing JSON documents based on logical expressions.
-data-management-json,jello,,https://github.com/kellyjonbrazil/jello,"CLI tool to filter JSON and JSON Lines data with Python syntax, similar to - suprise :-), jq!"
+data-management-json,jello,,https://github.com/kellyjonbrazil/jello,"CLI tool to filter JSON and JSON Lines data with Python syntax, similar to - surprise :-), jq!"
```
Comment (vitorhcl) [inline data/apps.csv:1128]:
```suggestion
data-management-json,jello,,https://github.com/kellyjonbrazil/jello,"Filter JSON and JSON Lines data with Python syntax, similar to - surprise :-), jq!"
```

File: data/apps.csv [hunk 4]
```diff
@@ -1386,7 +1386,7 @@ data-management-tabular,textql,,https://github.com/dinedal/textql,Execute SQL ag
 utility,tickrs,,https://github.com/tarkah/tickrs,Realtime ticker data in your terminal.
 chat,toot,,https://github.com/ihabunek/toot,Mastodon CLI & TUI.
 package-manager,topgrade,,https://github.com/topgrade-rs/topgrade,Upgrade all the things.
-networking,tproxy,,https://github.com/kevwan/tproxy,A cli tool to proxy and analyze TCP connections.
+networking,tproxy,,https://github.com/kevwan/tproxy,A CLI tool to proxy and analyze TCP connections.
```
Comment (vitorhcl) [inline data/apps.csv:1389]:
```suggestion
networking,tproxy,,https://github.com/kevwan/tproxy,Proxy and analyze TCP connections.
```

File: data/apps.csv [hunk 5]
```diff
@@ -1239,16 +1239,16 @@ git,mergestat-lite,,https://github.com/mergestat/mergestat-lite,A command-line t
 git,git-quick-stats,,https://github.com/arzzen/git-quick-stats,A simple and efficient way to access various statistics in a git repository.
 git,git-recall,,https://github.com/Fakerr/git-recall,A simple tool that allows you to easily go through your commits and check what you or other contributors in your team did.
 networking,goaccess,,https://github.com/allinurl/goaccess,"GoAccess is a real-time web log analyzer and interactive viewer, that provides fast and valuable HTTP statistics."
-networking,wuzz,,https://github.com/asciimoo/wuzz,Interactive cli tool for HTTP inspection.
+networking,wuzz,,https://github.com/asciimoo/wuzz,Interactive CLI tool for HTTP inspection.
```
Comment (vitorhcl) [inline data/apps.csv:1242]:
```suggestion
networking,wuzz,,https://github.com/asciimoo/wuzz,Interactively inspect or modify HTTP requests.
```

File: data/apps.csv [hunk 6]
```diff
@@ -986,7 +986,7 @@ system,killport,,https://github.com/jkfran/killport,A command-line tool to easil
 copilot,CLI Co-Pilot,,https://github.com/AntonOsika/CLI-Co-Pilot,CLI tool that uses GPT4 to turn natural language commands into their Bash/ZShell/PowerShell equivalents.
 time-tracker,Timer-CLI,,https://github.com/1Blademaster/timer-cli,A very simple countdown timer.
 copilot,codemancer,https://0xmmo.github.io/codemancer/,https://github.com/0xmmo/codemancer,Code with GPT-4 from your command line.
-text-search,ast-grep,,https://github.com/ast-grep/ast-grep,"A CLI tool for code structrual search, lint and rewriting."
+text-search,ast-grep,,https://github.com/ast-grep/ast-grep,"A CLI tool for code structural search, lint and rewriting."
```
Comment (vitorhcl) [inline data/apps.csv:989]:
```suggestion
text-search,ast-grep,,https://github.com/ast-grep/ast-grep,"Code structural search, linting and rewriting."
```

File: data/apps.csv [hunk 7]
```diff
@@ -788,13 +788,13 @@ cheatsheet,MUC,,https://github.com/nate-sys/muc,Visualize your most used command
...
-security,PaperAge,,https://github.com/matiaskorhonen/paper-age,"Easy and secure paper backups of secrets, which takes a text and generates an encrypted QRcode to print on paper."
+security,PaperAge,,https://github.com/matiaskorhonen/paper-age,"Easy and secure paper backups of secrets, which takes a text and generates an encrypted QR code to print on paper."
 utility,Autocomplete,,https://github.com/withfig/autocomplete,IDE-style autocomplete for your existing terminal & shell.
 ai,chatgpt,,https://github.com/mglantz/chatgpt,Simple command line integration to Chat GPT.
 organizers,Calcure,,https://github.com/anufrievroman/calcure,Modern TUI calendar and task manager with customizable interface.
 rss,Canto Curses,,https://github.com/themoken/canto-curses,Curses frontend for [Canto daemon](https://github.com/themoken/canto-next) for RSS feeds.
 rss,rReader,,https://github.com/rainygirl/rreader,RSS reader client with TUI interface.
-rss,rss-cli,,https://github.com/Clortox/rss-cli,A unix-inspired cli application for interacting with RSS feeds.
+rss,rss-cli,,https://github.com/Clortox/rss-cli,A UNIX-inspired CLI application for interacting with RSS feeds.
```
Comment (vitorhcl) [inline data/apps.csv:797]:
```suggestion
rss,rss-cli,,https://github.com/Clortox/rss-cli,A UNIX-inspired application for interacting with RSS feeds.
```

File: data/articles.csv [hunk 8]
```diff
@@ -11,16 +11,16 @@ WordGrinder,WordGrinder: Distraction-Free Writing From the Command Line,http://f
...
-OnionShare,OnionShare - A Tool To Share Any Size Of Files Securely And Anonymously,https://www.2daygeek.com/onionshare-secure-way-to-share-files-sharing-tool-linux/,2017-12-04,"When we wrote an article about file sharing utilities such as Magic Womhole & transfer.sh most of the users asked is it secure? and where the files are stored.  So, we decided to bring one of the best utility that stores and share files securely over the Internet. When we dig on that, we came to know about OnionShare which fulfill all this requirements."
+OnionShare,OnionShare - A Tool To Share Any Size Of Files Securely And Anonymously,https://www.2daygeek.com/onionshare-secure-way-to-share-files-sharing-tool-linux/,2017-12-04,"When we wrote an article about file sharing utilities such as Magic Womhole & transfer.sh most of the users asked is it secure? and where the files are stored. So, we decided to bring one of the best utility that stores and share files securely over the Internet. When we dig on that, we came to know about OnionShare which fulfill all this requirements."
```
Comment (vitorhcl) [inline data/articles.csv:19]:
```suggestion
OnionShare,Share files of any size securely and anonymously,https://www.2daygeek.com/onionshare-secure-way-to-share-files-sharing-tool-linux/,2017-12-04,"When we wrote an article about file sharing utilities such as Magic Womhole & transfer.sh most of the users asked is it secure? and where the files are stored. So, we decided to bring one of the best utility that stores and share files securely over the Internet. When we dig on that, we came to know about OnionShare which fulfill all this requirements."
```
We don't need to leave the project name twice.

File: data/apps.csv (hunk 1 above)
Comment (ccoVeille) [reply data/apps.csv:432]: I'm unsure about these changes

File: data/apps.csv (hunk 2 above)
Comment (ccoVeille) [reply data/apps.csv:1103]: I'm unsure about these changes

**Comment by ccoVeille** [comment]
@vitorhcl I'm open to the changes you suggest., I would prefer to keep them separate from this PR. So, I resolved.

I hope you will understand

**Comment by ccoVeille** [comment]
I mean I intended this PR to be about typo and fix, not about rewording.

**Comment by vitorhcl** [comment]
Ok, I understand.

**Comment by ccoVeille** [comment]
The main reason is that with such amount of changes I'm doing and the fact the rewording could be iterative and take time.

I will would like to avoid possible conflicts if other PR are merged in the meantime

File: data/articles.csv [hunk 9]
```diff
@@ -11,16 +11,16 @@ WordGrinder,WordGrinder: Distraction-Free Writing From the Command Line,http://f
...
+lshw,LSHW (Hardware Lister) – A Nifty Tool To Get A Hardware Information On Linux,https://www.2daygeek.com/lshw-find-check-system-hardware-information-details-linux/,2017-10-07,"lshw (stands for Hardware Lister) is a small nifty tool that generates detailed reports about various hardware components on the machine such as memory configuration, firmware version, mainboard configuration, CPU version and speed, cache configuration, USB, network card, graphics cards, multimedia, printers, bus speed, etc."
+titan,A Command line Password Manager For Linux,https://www.ostechnix.com/titan-command-line-password-manager-linux/,2017-07-10,There are plethora of GUI based password managers. A quick google search will lead you to pick one suitable for you. But the command line password managers are very few. The one today we talk about is Titan. It is written using C programming language. It uses SQlite.
```
Comment (vitorhcl) [inline data/articles.csv:15]:
```suggestion
titan,A Command line Password Manager For Linux,https://www.ostechnix.com/titan-command-line-password-manager-linux/,2017-07-10,There are plethora of GUI based password managers. A quick Google search will lead you to pick one suitable for you. But the command line password managers are very few. The one today we talk about is Titan. It is written using C programming language. It uses SQlite.
```

File: data/articles.csv [hunk 10]
```diff
@@ -11,16 +11,16 @@ WordGrinder,WordGrinder: Distraction-Free Writing From the Command Line,http://f
...
-socli,SoCLI - A Tool to Search Stack Overflow from Linux Terminal,https://linoxide.com/linux-how-to/socli-tool-access-stack-overflow-linux-terminal/,2017-10-07,"In this article, I am going to show you how to install Stack Overflow command line interface (SoCLI) on Linux.  Stack Overflow is a question and answer website on the topic of computer programming.  We can use command line interface to get any information from Linux terminal without using the graphical web interface or terminal browser."
+socli,SoCLI - A Tool to Search Stack Overflow from Linux Terminal,https://linoxide.com/linux-how-to/socli-tool-access-stack-overflow-linux-terminal/,2017-10-07,"In this article, I am going to show you how to install Stack Overflow command line interface (SoCLI) on Linux. Stack Overflow is a question and answer website on the topic of computer programming. We can use command line interface to get any information from Linux terminal without using the graphical web interface or terminal browser."
```
Comment (vitorhcl) [inline data/articles.csv:26]:
```suggestion
socli,SoCLI - A Tool to Search Stack Overflow from Linux Terminal,https://linoxide.com/linux-how-to/socli-tool-access-stack-overflow-linux-terminal/,2017-10-07,"In this article, I am going to show you how to install Stack Overflow command line interface (SoCLI) on Linux. StackOverflow is a question and answer website on the topic of computer programming. We can use command line interface to get any information from Linux terminal without using the graphical web interface or terminal browser."
```

File: data/categories.csv [hunk 11]
```diff
@@ -29,7 +29,7 @@ file-system,File systems,"File systems with specific features; e.g., the possibi
 financial,Financial tools,"Personal ledger trackers, currency converters, and tools to manage and track cryptocurrencies"
 flashcard,"Anki, decks and flashcards",Manage decks of flashcards and Anki decks
 font,Font management,Utilities to manage system fonts and to generate text using ASCII-art-like characters
-funny,Funny tools,"Miscellaneous of tools that provide some funny/aesthetical functionality (animations, funny quotes, original message visualization, etc.)"
+funny,Funny tools,"Miscellaneous of tools that provide some funny/esthetical functionality (animations, funny quotes, original message visualization, etc.)"
```
Comment (vitorhcl) [inline data/categories.csv:32]:
```suggestion
funny,Funny tools,"Miscellaneous of tools that provide some funny/aesthetical functionality (animations, funny quotes, original message visualization, etc.)"
```
This one was actually right

**Review by vitorhcl — CHANGES_REQUESTED** [review]
My suggestions are about wrong words now, and not rewording

File: data/articles.csv (hunk 9 above)
Comment (ccoVeille) [reply data/articles.csv:15]: Good catch

File: data/articles.csv (hunk 10 above)
Comment (ccoVeille) [reply data/articles.csv:26]: I would also change the other one on the line and will check for others in the repository

Comment (vitorhcl) [reply data/articles.csv:26]: This was right before

File: data/articles.csv [hunk 12]
```diff
@@ -11,16 +11,16 @@ WordGrinder,WordGrinder: Distraction-Free Writing From the Command Line,http://f
...
-lolcat,lolcat : tool to rainbow color linux terminal,http://kerneltalks.com/linux/lolcat-tool-to-rainbow-color-linux-terminal/,2017-07-10,Paint your command outputs with rainbow color!  Use lolcat (Ruby gem) tool and add some spice to black putty terminal!
+lolcat,lolcat : tool to rainbow color linux terminal,http://kerneltalks.com/linux/lolcat-tool-to-rainbow-color-linux-terminal/,2017-07-10,Paint your command outputs with rainbow color! Use lolcat (Ruby gem) tool and add some spice to black putty terminal!
 neofetch,Display Linux Distribution Logo in ASCII Art in Terminal,https://itsfoss.com/display-linux-logo-in-ascii/,2017-10-07,... You might have seen people sharing such desktop screenshot with terminal showing the logo of the Linux distribution used in ASCII art.
 pdd,pdd - Tool to find Date and Time Difference in Linux Command Line,https://linoxide.com/linux-how-to/pdd-tool-date-time-difference-linux-command-line/,2017-08-09,"In some occasions where you want to check by how many years someone older than you, how old you are (in days, years or months), the countdown to an event or the next flash sale. There is a python-based command line application ..."
```
Comment (vitorhcl) [inline data/articles.csv:25]:
```suggestion
 pdd,pdd - Tool to find Date and Time Difference in Linux Command Line,https://linoxide.com/linux-how-to/pdd-tool-date-time-difference-linux-command-line/,2017-08-09,"In some occasions where you want to check by how many years someone older than you, how old you are (in days, years or months), the countdown to an event or the next flash sale. There is a Python-based command line application ..."
```
Maybe there are other cases of "python"?

Comment (ccoVeille) [reply data/articles.csv:25]: I'll check, thanks

File: data/categories.csv (hunk 11 above)
Comment (ccoVeille) [reply data/categories.csv:32]: I rollbacked this one

**Comment by ccoVeille** [comment]
everything should be ok now @vitorhcl

**Comment by ccoVeille** [comment]
I found many other errors I fixed by checking again

**Comment by toolleeo** [comment]
Thanks for the contribution @ccoVeille .
It was a demanding pull request, but very useful.

**Comment by ccoVeille** [comment]
> Thanks for the contribution @ccoVeille . It was a demanding pull request, but very useful.

Yes, even I helped myself with tools such as [languagetool](https://github.com/languagetool-org/languagetool), [vale](https://github.com/errata-ai/vale), [codespell](https://github.com/codespell-project/codespell), and [typos](https://github.com/crate-ci/typos)

I had to read it again and again to make sure everything was ok.

**Pull request merged**